src/main/java/com/google/sps/servlets/SignInServlet.java

```diff
@@ -44,24 +43,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 		Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
 		 
 		// Get user email from server
-	        String userEmail = getUserField(username, password, datastore, "email");
-	        String name = getUserField(username, password, datastore, "name");
+	        String userEmail = getStringField(username, password, datastore, "email");
+	        String name = getStringField(username, password, datastore, "name");
+	        String appearingOnline = getBooleanField(username, password, datastore, 
```

Comment:
Does it need to be converted into a String?

Reply:
Java arrays can only have one type. It was easier to send a boolean as a string and then check on frontend then creating another class with a "String userEmail; String name; boolean appearingOnline" and sending that as json to the user.